imports/api/tasks/tasks.js

```diff
@@ -1,3 +1,17 @@
 import { Mongo } from 'meteor/mongo';
 
 export const Tasks = new Mongo.Collection('tasks');
+
+var Schemas = {};
+
+Schemas.Tasks = new SimpleSchema({
+  name: { type: String },
+  completed_on: {type: Date, optional: true},
+  created_on: {type: Date},
+  due_on: { type: Date },
+  is_urgent: { type: Boolean},
+  is_important: { type: Boolean}
```

Comment:
Can you default these both to false?
Hmmm not sure how to do this actually. How do we differentiate between unsorted and not urgent + not important? Using nulls feels kind of unsafe